Conda build failing on MacOS

```
[ 76%] Building CXX object src/CMakeFiles/xmipp4-core.dir/multidimensional/array_factory.cpp.o
  In file included from /Users/runner/miniconda3/conda-bld/xmipp4-core_1774390577388/work/src/multidimensional/array_factory.cpp:3:
  In file included from /Users/runner/miniconda3/conda-bld/xmipp4-core_1774390577388/work/include/xmipp4/core/multidimensional/array_factory.hpp:6:
  In file included from /Users/runner/miniconda3/conda-bld/xmipp4-core_1774390577388/work/include/xmipp4/core/multidimensional/../scalar_ref.hpp:72:
  In file included from /Users/runner/miniconda3/conda-bld/xmipp4-core_1774390577388/work/include/xmipp4/core/multidimensional/../scalar_ref.inl:5:
  In file included from /Users/runner/miniconda3/conda-bld/xmipp4-core_1774390577388/work/include/xmipp4/core/multidimensional/../numerical_type_traits.hpp:6:
  In file included from /Users/runner/miniconda3/conda-bld/xmipp4-core_1774390577388/work/include/xmipp4/core/multidimensional/../fixed_float.hpp:5:
  /tmp/tmpp2ugx_wu/build/_deps/half-src/include/half.hpp:445:20: warning: identifier '_h' preceded by whitespace in a literal operator declaration is deprecated [-Wdeprecated-literal-operator]
    445 |                 half operator "" _h(long double);
        |                      ~~~~~~~~~~~~^~
        |                      operator""_h
  /tmp/tmpp2ugx_wu/build/_deps/half-src/include/half.hpp:2264:36: warning: identifier '_h' preceded by whitespace in a literal operator declaration is deprecated [-Wdeprecated-literal-operator]
   2264 |                 friend half literal::operator "" _h(long double);
        |                                      ~~~~~~~~~~~~^~
        |                                      operator""_h
  /tmp/tmpp2ugx_wu/build/_deps/half-src/include/half.hpp:2279:27: warning: identifier '_h' preceded by whitespace in a literal operator declaration is deprecated [-Wdeprecated-literal-operator]
   2279 |                 inline half operator "" _h(long double value) { return half(detail::binary, detail::float2half<half::round_style>(value)); }
        |                             ~~~~~~~~~~~~^~
        |                             operator""_h
  3 warnings generated.
  [ 77%] Building CXX object src/CMakeFiles/xmipp4-core.dir/multidimensional/array_iterator.cpp.o
  [ 78%] Building CXX object src/CMakeFiles/xmipp4-core.dir/multidimensional/array_view.cpp.o
  [ 79%] Building CXX object src/CMakeFiles/xmipp4-core.dir/multidimensional/broadcast.cpp.o
  [ 80%] Building CXX object src/CMakeFiles/xmipp4-core.dir/multidimensional/broadcast_error.cpp.o
  [ 81%] Building CXX object src/CMakeFiles/xmipp4-core.dir/multidimensional/eager_operation_dispatcher.cpp.o
  In file included from /Users/runner/miniconda3/conda-bld/xmipp4-core_1774390577388/work/src/multidimensional/eager_operation_dispatcher.cpp:9:
  In file included from /Users/runner/miniconda3/conda-bld/xmipp4-core_1774390577388/work/include/xmipp4/core/multidimensional/array_factory.hpp:6:
  In file included from /Users/runner/miniconda3/conda-bld/xmipp4-core_1774390577388/work/include/xmipp4/core/multidimensional/../scalar_ref.hpp:72:
  In file included from /Users/runner/miniconda3/conda-bld/xmipp4-core_1774390577388/work/include/xmipp4/core/multidimensional/../scalar_ref.inl:5:
  In file included from /Users/runner/miniconda3/conda-bld/xmipp4-core_1774390577388/work/include/xmipp4/core/multidimensional/../numerical_type_traits.hpp:6:
  In file included from /Users/runner/miniconda3/conda-bld/xmipp4-core_1774390577388/work/include/xmipp4/core/multidimensional/../fixed_float.hpp:5:
  /tmp/tmpp2ugx_wu/build/_deps/half-src/include/half.hpp:445:20: warning: identifier '_h' preceded by whitespace in a literal operator declaration is deprecated [-Wdeprecated-literal-operator]
    445 |                 half operator "" _h(long double);
        |                      ~~~~~~~~~~~~^~
        |                      operator""_h
  /tmp/tmpp2ugx_wu/build/_deps/half-src/include/half.hpp:2264:36: warning: identifier '_h' preceded by whitespace in a literal operator declaration is deprecated [-Wdeprecated-literal-operator]
   2264 |                 friend half literal::operator "" _h(long double);
        |                                      ~~~~~~~~~~~~^~
        |                                      operator""_h
  /tmp/tmpp2ugx_wu/build/_deps/half-src/include/half.hpp:2279:27: warning: identifier '_h' preceded by whitespace in a literal operator declaration is deprecated [-Wdeprecated-literal-operator]
   2279 |                 inline half operator "" _h(long double value) { return half(detail::binary, detail::float2half<half::round_style>(value)); }
        |                             ~~~~~~~~~~~~^~
        |                             operator""_h
  3 warnings generated.
  [ 82%] Building CXX object src/CMakeFiles/xmipp4-core.dir/multidimensional/index.cpp.o
  [ 83%] Building CXX object src/CMakeFiles/xmipp4-core.dir/multidimensional/kernel.cpp.o
  [ 84%] Building CXX object src/CMakeFiles/xmipp4-core.dir/multidimensional/kernel_builder.cpp.o
  [ 86%] Building CXX object src/CMakeFiles/xmipp4-core.dir/multidimensional/kernel_manager.cpp.o
  [ 87%] Building CXX object src/CMakeFiles/xmipp4-core.dir/multidimensional/operation.cpp.o
  [ 88%] Building CXX object src/CMakeFiles/xmipp4-core.dir/multidimensional/operation_dispatcher.cpp.o
  [ 89%] Building CXX object src/CMakeFiles/xmipp4-core.dir/multidimensional/operations/copy_operation.cpp.o
  [ 90%] Building CXX object src/CMakeFiles/xmipp4-core.dir/multidimensional/operations/fill_operation.cpp.o
  In file included from /Users/runner/miniconda3/conda-bld/xmipp4-core_1774390577388/work/src/multidimensional/operations/fill_operation.cpp:3:
  In file included from /Users/runner/miniconda3/conda-bld/xmipp4-core_1774390577388/work/include/xmipp4/core/multidimensional/operations/fill_operation.hpp:6:
  In file included from /Users/runner/miniconda3/conda-bld/xmipp4-core_1774390577388/work/include/xmipp4/core/multidimensional/operations/../../scalar_ref.hpp:72:
  In file included from /Users/runner/miniconda3/conda-bld/xmipp4-core_1774390577388/work/include/xmipp4/core/multidimensional/operations/../../scalar_ref.inl:5:
  In file included from /Users/runner/miniconda3/conda-bld/xmipp4-core_1774390577388/work/include/xmipp4/core/multidimensional/operations/../../numerical_type_traits.hpp:6:
  In file included from /Users/runner/miniconda3/conda-bld/xmipp4-core_1774390577388/work/include/xmipp4/core/multidimensional/operations/../../fixed_float.hpp:5:
  /tmp/tmpp2ugx_wu/build/_deps/half-src/include/half.hpp:445:20: warning: identifier '_h' preceded by whitespace in a literal operator declaration is deprecated [-Wdeprecated-literal-operator]
    445 |                 half operator "" _h(long double);
        |                      ~~~~~~~~~~~~^~
        |                      operator""_h
  /tmp/tmpp2ugx_wu/build/_deps/half-src/include/half.hpp:2264:36: warning: identifier '_h' preceded by whitespace in a literal operator declaration is deprecated [-Wdeprecated-literal-operator]
   2264 |                 friend half literal::operator "" _h(long double);
        |                                      ~~~~~~~~~~~~^~
        |                                      operator""_h
  /tmp/tmpp2ugx_wu/build/_deps/half-src/include/half.hpp:2279:27: warning: identifier '_h' preceded by whitespace in a literal operator declaration is deprecated [-Wdeprecated-literal-operator]
   2279 |                 inline half operator "" _h(long double value) { return half(detail::binary, detail::float2half<half::round_style>(value)); }
        |                             ~~~~~~~~~~~~^~
        |                             operator""_h
  3 warnings generated.
  [ 91%] Building CXX object src/CMakeFiles/xmipp4-core.dir/multidimensional/strided_layout.cpp.o
  [ 92%] Building CXX object src/CMakeFiles/xmipp4-core.dir/numerical_type.cpp.o
  In file included from /Users/runner/miniconda3/conda-bld/xmipp4-core_1774390577388/work/src/numerical_type.cpp:5:
  In file included from /Users/runner/miniconda3/conda-bld/xmipp4-core_1774390577388/work/include/xmipp4/core/fixed_float.hpp:5:
  /tmp/tmpp2ugx_wu/build/_deps/half-src/include/half.hpp:445:20: warning: identifier '_h' preceded by whitespace in a literal operator declaration is deprecated [-Wdeprecated-literal-operator]
    445 |                 half operator "" _h(long double);
        |                      ~~~~~~~~~~~~^~
        |                      operator""_h
  /tmp/tmpp2ugx_wu/build/_deps/half-src/include/half.hpp:2264:36: warning: identifier '_h' preceded by whitespace in a literal operator declaration is deprecated [-Wdeprecated-literal-operator]
   2264 |                 friend half literal::operator "" _h(long double);
        |                                      ~~~~~~~~~~~~^~
        |                                      operator""_h
  /tmp/tmpp2ugx_wu/build/_deps/half-src/include/half.hpp:2279:27: warning: identifier '_h' preceded by whitespace in a literal operator declaration is deprecated [-Wdeprecated-literal-operator]
   2279 |                 inline half operator "" _h(long double value) { return half(detail::binary, detail::float2half<half::round_style>(value)); }
        |                             ~~~~~~~~~~~~^~
        |                             operator""_h
  3 warnings generated.
  [ 93%] Building CXX object src/CMakeFiles/xmipp4-core.dir/plugin_loader.cpp.o
  [ 94%] Building CXX object src/CMakeFiles/xmipp4-core.dir/plugin_manager.cpp.o
  [ 95%] Building CXX object src/CMakeFiles/xmipp4-core.dir/service_catalog.cpp.o
  [ 96%] Building CXX object src/CMakeFiles/xmipp4-core.dir/system/dynamic_library.cpp.o
  [ 97%] Building CXX object src/CMakeFiles/xmipp4-core.dir/system/host.cpp.o
  [ 98%] Building CXX object src/CMakeFiles/xmipp4-core.dir/system/memory_mapped_file.cpp.o
  [100%] Linking CXX shared library libxmipp4-core.dylib
  Undefined symbols for architecture arm64:
    "std::__1::__hash_memory(void const*, unsigned long)", referenced from:
        std::__1::__hash_const_iterator<std::__1::__hash_node<std::__1::__hash_value_type<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::unique_ptr<xmipp4::communication::device_communicator_backend, std::__1::default_delete<xmipp4::communication::device_communicator_backend>>>, void*>*> std::__1::__hash_table<std::__1::__hash_value_type<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::unique_ptr<xmipp4::communication::device_communicator_backend, std::__1::default_delete<xmipp4::communication::device_communicator_backend>>>, std::__1::__unordered_map_hasher<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const, std::__1::unique_ptr<xmipp4::communication::device_communicator_backend, std::__1::default_delete<xmipp4::communication::device_communicator_backend>>>, std::__1::hash<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>, std::__1::equal_to<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>, true>, std::__1::__unordered_map_equal<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const, std::__1::unique_ptr<xmipp4::communication::device_communicator_backend, std::__1::default_delete<xmipp4::communication::device_communicator_backend>>>, std::__1::equal_to<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>, std::__1::hash<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>, true>, std::__1::allocator<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const, std::__1::unique_ptr<xmipp4::communication::device_communicator_backend, std::__1::default_delete<xmipp4::communication::device_communicator_backend>>>>>::find<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&) const in device_communicator_manager.cpp.o
        std::__1::pair<std::__1::__hash_iterator<std::__1::__hash_node<std::__1::__hash_value_type<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::unique_ptr<xmipp4::communication::device_communicator_backend, std::__1::default_delete<xmipp4::communication::device_communicator_backend>>>, void*>*>, bool> std::__1::__hash_table<std::__1::__hash_value_type<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::unique_ptr<xmipp4::communication::device_communicator_backend, std::__1::default_delete<xmipp4::communication::device_communicator_backend>>>, std::__1::__unordered_map_hasher<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const, std::__1::unique_ptr<xmipp4::communication::device_communicator_backend, std::__1::default_delete<xmipp4::communication::device_communicator_backend>>>, std::__1::hash<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>, std::__1::equal_to<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>, true>, std::__1::__unordered_map_equal<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const, std::__1::unique_ptr<xmipp4::communication::device_communicator_backend, std::__1::default_delete<xmipp4::communication::device_communicator_backend>>>, std::__1::equal_to<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>, std::__1::hash<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>, true>, std::__1::allocator<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const, std::__1::unique_ptr<xmipp4::communication::device_communicator_backend, std::__1::default_delete<xmipp4::communication::device_communicator_backend>>>>>::__emplace_unique_key_args<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::unique_ptr<xmipp4::communication::device_communicator_backend, std::__1::default_delete<xmipp4::communication::device_communicator_backend>>>(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>&&, std::__1::unique_ptr<xmipp4::communication::device_communicator_backend, std::__1::default_delete<xmipp4::communication::device_communicator_backend>>&&) in device_communicator_manager.cpp.o
        std::__1::__hash_const_iterator<std::__1::__hash_node<std::__1::__hash_value_type<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::unique_ptr<xmipp4::communication::host_communicator_backend, std::__1::default_delete<xmipp4::communication::host_communicator_backend>>>, void*>*> std::__1::__hash_table<std::__1::__hash_value_type<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::unique_ptr<xmipp4::communication::host_communicator_backend, std::__1::default_delete<xmipp4::communication::host_communicator_backend>>>, std::__1::__unordered_map_hasher<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const, std::__1::unique_ptr<xmipp4::communication::host_communicator_backend, std::__1::default_delete<xmipp4::communication::host_communicator_backend>>>, std::__1::hash<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>, std::__1::equal_to<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>, true>, std::__1::__unordered_map_equal<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const, std::__1::unique_ptr<xmipp4::communication::host_communicator_backend, std::__1::default_delete<xmipp4::communication::host_communicator_backend>>>, std::__1::equal_to<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>, std::__1::hash<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>, true>, std::__1::allocator<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const, std::__1::unique_ptr<xmipp4::communication::host_communicator_backend, std::__1::default_delete<xmipp4::communication::host_communicator_backend>>>>>::find<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&) const in host_communicator_manager.cpp.o
        std::__1::pair<std::__1::__hash_iterator<std::__1::__hash_node<std::__1::__hash_value_type<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::unique_ptr<xmipp4::communication::host_communicator_backend, std::__1::default_delete<xmipp4::communication::host_communicator_backend>>>, void*>*>, bool> std::__1::__hash_table<std::__1::__hash_value_type<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::unique_ptr<xmipp4::communication::host_communicator_backend, std::__1::default_delete<xmipp4::communication::host_communicator_backend>>>, std::__1::__unordered_map_hasher<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const, std::__1::unique_ptr<xmipp4::communication::host_communicator_backend, std::__1::default_delete<xmipp4::communication::host_communicator_backend>>>, std::__1::hash<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>, std::__1::equal_to<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>, true>, std::__1::__unordered_map_equal<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const, std::__1::unique_ptr<xmipp4::communication::host_communicator_backend, std::__1::default_delete<xmipp4::communication::host_communicator_backend>>>, std::__1::equal_to<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>, std::__1::hash<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>, true>, std::__1::allocator<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const, std::__1::unique_ptr<xmipp4::communication::host_communicator_backend, std::__1::default_delete<xmipp4::communication::host_communicator_backend>>>>>::__emplace_unique_key_args<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::unique_ptr<xmipp4::communication::host_communicator_backend, std::__1::default_delete<xmipp4::communication::host_communicator_backend>>>(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>&&, std::__1::unique_ptr<xmipp4::communication::host_communicator_backend, std::__1::default_delete<xmipp4::communication::host_communicator_backend>>&&) in host_communicator_manager.cpp.o
        std::__1::pair<std::__1::__hash_iterator<std::__1::__hash_node<std::__1::__hash_value_type<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::shared_ptr<spdlog::logger>>, void*>*>, bool> std::__1::__hash_table<std::__1::__hash_value_type<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::shared_ptr<spdlog::logger>>, std::__1::__unordered_map_hasher<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const, std::__1::shared_ptr<spdlog::logger>>, std::__1::hash<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>, std::__1::equal_to<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>, true>, std::__1::__unordered_map_equal<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const, std::__1::shared_ptr<spdlog::logger>>, std::__1::equal_to<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>, std::__1::hash<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>, true>, std::__1::allocator<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const, std::__1::shared_ptr<spdlog::logger>>>>::__emplace_unique_key_args<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::piecewise_construct_t const&, std::__1::tuple<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>&&>, std::__1::tuple<>>(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, std::__1::piecewise_construct_t const&, std::__1::tuple<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>&&>&&, std::__1::tuple<>&&) in caching_memory_allocator.cpp.o
        xmipp4::hardware::memory_block_pool::release(xmipp4::hardware::memory_heap const&) in memory_block_pool.cpp.o
        std::__1::pair<std::__1::__hash_iterator<std::__1::__hash_node<std::__1::__hash_value_type<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::shared_ptr<spdlog::logger>>, void*>*>, bool> std::__1::__hash_table<std::__1::__hash_value_type<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::shared_ptr<spdlog::logger>>, std::__1::__unordered_map_hasher<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const, std::__1::shared_ptr<spdlog::logger>>, std::__1::hash<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>, std::__1::equal_to<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>, true>, std::__1::__unordered_map_equal<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const, std::__1::shared_ptr<spdlog::logger>>, std::__1::equal_to<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>, std::__1::hash<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>>, true>, std::__1::allocator<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const, std::__1::shared_ptr<spdlog::logger>>>>::__emplace_unique_key_args<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>, std::__1::piecewise_construct_t const&, std::__1::tuple<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>&&>, std::__1::tuple<>>(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>> const&, std::__1::piecewise_construct_t const&, std::__1::tuple<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char>>&&>&&, std::__1::tuple<>&&) in memory_block_pool.cpp.o
        ...
  ld: symbol(s) not found for architecture arm64
  arm64-apple-darwin20.0: error: linker command failed with exit code 1 (use -v to see invocation)
  make[2]: *** [src/CMakeFiles/xmipp4-core.dir/build.make:1192: src/libxmipp4-core.0.1.0.dylib] Error 1
  make[1]: *** [CMakeFiles/Makefile2:1187: src/CMakeFiles/xmipp4-core.dir/all] Error 2
  make: *** [Makefile:136: all] Error 2

  *** CMake build failed
  error: subprocess-exited-with-error
  
  × Building wheel for xmipp4-core (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> No available output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  full command: /Users/runner/miniconda3/conda-bld/xmipp4-core_1774390577388/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/bin/python /Users/runner/miniconda3/conda-bld/xmipp4-core_1774390577388/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho/lib/python3.12/site-packages/pip/_vendor/pyproject_hooks/_in_process/_in_process.py build_wheel /tmp/tmpvj3e61l3
  cwd: /Users/runner/miniconda3/conda-bld/xmipp4-core_1774390577388/work
  ERROR: Failed building wheel for xmipp4-core
error: failed-wheel-build-for-install
```

--- a/.github/workflows/build-and-test.yml
+++ b/.github/workflows/build-and-test.yml
@@ -129,7 +129,7 @@ jobs:
       matrix:
         os:
         - ubuntu-latest
-        #- macos-latest # FIXME
+        - macos-latest
         - windows-latest
 
     runs-on: ${{ matrix.os }}

--- a/conda/meta.yaml
+++ b/conda/meta.yaml
@@ -27,6 +27,7 @@ build:
     # C++17 features marked as not available
     # https://conda-forge.org/docs/maintainer/knowledge_base.html#newer-c-features-with-old-sdk
     - export CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"  # [osx]
+    - export MACOSX_DEPLOYMENT_TARGET=13  # [osx]
     
     # Make sure an existing Visual Studio installation is being used 
     - "powershell -NoProfile -NonInteractive -ExecutionPolicy Bypass -File \"%RECIPE_DIR%\\visual-studio-name-detector.ps1\" > generator.txt"  # [win]

--- a/conda/meta.yaml
+++ b/conda/meta.yaml
@@ -52,4 +52,5 @@ requirements:
   host:
     - python
     - pip
+    - libcxx
     - scikit-build-core>=0.10

--- a/conda/meta.yaml
+++ b/conda/meta.yaml
@@ -52,5 +52,5 @@ requirements:
   host:
     - python
     - pip
-    - libcxx
+    - libcxx # [osx]
     - scikit-build-core>=0.10

--- a/conda/meta.yaml
+++ b/conda/meta.yaml
@@ -24,10 +24,6 @@ build:
   script_env:
     - XMIPP4_CORE_BUILD_SPDLOG=OFF
   script:
-    # C++17 features marked as not available
-    # https://conda-forge.org/docs/maintainer/knowledge_base.html#newer-c-features-with-old-sdk
-    # - export CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"  # [osx]
-    
     # Make sure an existing Visual Studio installation is being used 
     - "powershell -NoProfile -NonInteractive -ExecutionPolicy Bypass -File \"%RECIPE_DIR%\\visual-studio-name-detector.ps1\" > generator.txt"  # [win]
     - "set /p VS_GENERATOR=<generator.txt"  # [win]

--- a/conda/meta.yaml
+++ b/conda/meta.yaml
@@ -26,8 +26,7 @@ build:
   script:
     # C++17 features marked as not available
     # https://conda-forge.org/docs/maintainer/knowledge_base.html#newer-c-features-with-old-sdk
-    - export CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"  # [osx]
-    - export MACOSX_DEPLOYMENT_TARGET=13  # [osx]
+    # - export CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"  # [osx]
     
     # Make sure an existing Visual Studio installation is being used 
     - "powershell -NoProfile -NonInteractive -ExecutionPolicy Bypass -File \"%RECIPE_DIR%\\visual-studio-name-detector.ps1\" > generator.txt"  # [win]
@@ -52,5 +51,4 @@ requirements:
   host:
     - python
     - pip
-    - libcxx # [osx]
     - scikit-build-core>=0.10

